### ISSUE TITLE
Add Documentation About [DEP0123] When Using IP's For Hostnames

### ISSUE THREAD

**Issue by Kyrluckechuck** [brianc/node-postgres#1950]
At present, if you provide an external IP for your hostname, upon connection you will receive a TLS deprecation warning as follows:
```
(node:18101) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an
  IP address is not permitted by RFC 6066. This will be ignored in a future 
  version.
```

It would appear that there is no way to mitigate this since it's a full deprecation introduced by the actual TLS libraries. Since many people who use this may be directly connecting to a server via IP and not always using a hostname (either modified in their `/etc/hosts` or a real subdomain/domain), they will likely encounter this and may spend quite a long time trying to resolve it.

I finally worked around it through a mixture of `/etc/hosts` entries and real DNS entries -- if this interpretation was wrong and there actually is a better solution that allows you to continue using IPs, please let me know! 

Just want to get this added to help others if it actually is the real solution and doesn't seem obvious to someone at my level.

**Comment by charmander** [comment]
The fix is in #1890.

**Comment by Kyrluckechuck** [comment]
Perfect. That is a much better solution to this and renders this useless, so I'll close it!
